daprdocs/content/en/reference/resource-specs/component-schema.md

```diff
@@ -32,6 +36,8 @@ spec:
 |--------------------|:--------:|---------|---------|
 | apiVersion         | Y        | The version of the Dapr (and Kubernetes if applicable) API you are calling | `dapr.io/v1alpha1`
 | kind               | Y        | The type of resource. For components is must always be `Component` | `Component`
+| auth               | N        | The authentication details for the component |
```

Comment:
```suggestion
| auth               | N        | The name of a secret store where `secretKeyRef` in the metadata lookup the name of secrets used in the component | `appsecretstore`
```

Reply:
See https://docs.dapr.io/operations/components/component-secrets/ for an example.